### PR TITLE
Use gs instead of geoserver for URLs in Docker

### PR DESCRIPTION
**Pull request by frafra**
WIP pull request for #4207.

### CODE REVIEW

File: scripts/docker/env/development/django.env [hunk 1]
```diff
@@ -1,7 +1,7 @@
 GEONODE_INSTANCE_NAME=geonode
 BROKER_URL=amqp://guest:guest@rabbitmq:5672/
 DJANGO_SETTINGS_MODULE=geonode.settings
-GEOSERVER_INTERNAL_URL=http://geoservices:8080/geoserver/
+GEOSERVER_INTERNAL_URL=http://geoservices:8080/gs/
```
Comment (afabiani) [inline scripts/docker/env/development/django.env]: Internal URLs (using port 8080) must be /geoserver/

**Comment by giohappy** [comment]
I think there's some confusion about the semantics of the locations.
**GEOSERVER_LOCATION**  as its internal URL to communicate with Geoserver. In Docker this should translate to the overlay network URL of the target service (geoserver).
**GEOSERVER_PUBLIC_LOCATION**  is instead the URL clients use to connect to Geoserver services (WMS, WFS, GWC, etc.). Since some commits ago this location points to the internal /gs proxy.
I need to review all the convoluted paths of these envs inside our compose files but this commit doesn't seem to fix the semantics mismatch.

**Comment by frafra** [comment]
Sorry, this was a WIP pull request; it seems that there still something to be changed, because in recent revisions I get a gateway timeout.

**Comment by frafra** [comment]
My pull request is broken. 
@giohappy could you please have a look at it and see if you can detect the error?

**Pull request merged**